openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -347,6 +347,15 @@ def sync_project_from_kitsu(dbcon: AvalonMongoDB, project: dict):
     if not project:
         project = gazu.project.get_project_by_name(project["name"])
 
+    # Get all statuses for projects from Kitsu
+    all_status = gazu.project.all_project_status()
+    for status in all_status:
+        if project['project_status_id'] == status['id']:
+            project['project_status_name'] = status['name']
+
+    if not get_project(project['name']) and project['project_status_name'] == "Closed":  # noqa
```

Comment:
I would recommend this order (the project is queried only if project status is closed and not all the time).
```suggestion
    if (
        project['project_status_name'] == "Closed"
        and not get_project(project['name'])
    ):
```